spec/system/task_spec.rb

```diff
@@ -1,27 +1,29 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
```

Comment:
`association`をFactoryに定義すると`project_id`の記載は不要になります。
```suggestion
  let(:task) { create(:task) }
```

Reply:
`ce23fb34df0bff4838dc6437ec19827086f2d615` でassociation を利用しました。